### PR TITLE
docs(CDK): add CDK Styling guide

### PR DESCRIPTION
**Pull request by kbo4sho**
Adds a guide for CDK styling.

### CODE REVIEW

File: guides/cdk-styling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Prebuilt CSS
+
+Though the CDK leaves most styling up to the developer, in some cases it is necessary to have styling to define or demonstrate a component. When necessary the CDK provides styles for this reason.
+
+### Overlay
+
+In order for the overlay in the CDK to work correctly you need to include the prebuilt overlay styles. 
+
+You can reference the styles in a SASS file like this.
```
Comment (donroyco) [inline guides/cdk-styling.md]: SASS » Sass, see also #9375

**Comment by jelbourn** [comment]
@kbo4sho Thanks for sending the PR. I'm thinking that this should actually be part of a more comprehensive guide on using the CDK in general. We're focused on getting 6.0 ready now, I'll come revisit this once that's released.

**Comment by kbo4sho** [comment]
@jelbourn I can see that making more sense for folks. When the time comes, I am happy to do more work on this. Thanks!

**Comment by jelbourn** [comment]
Closing this since we've since added docs for the cdk Sass (a11y and overlay) to their respective overviews.

**Pull request closed without merging**